package.json

```diff
@@ -17,9 +17,10 @@
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.0.9",
     "@brainhubeu/react-carousel": "1.19.26",
-    "@material-ui/core": "^4.12.3",
-    "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.57",
```

Comment:
Skeleton is the only thing in lab we're using, and it's in @mui/material in v5. So lab is gone